app/views/hosts/_bmc_missing_proxy.html.erb

```diff
@@ -0,0 +1,14 @@
+<div>
+<%
+  text = ''.html_safe
+  if bmc_count.zero?
+    text += _('No %s with BMC feature found, install one, enable BMC module and register it.').html_safe %
+      "<a href=\"/smart_proxies\">#{_('Smart Proxy')}</a>".html_safe
```

Comment:
Don't we have a URL mechanism for the smart proxies URL?

Reply:
Yeah I am just terrible Rails dev.